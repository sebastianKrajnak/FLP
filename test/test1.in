S,A,B
a,b,c,d
S
S->#
S->AB
A->aAb
A->ab
B->cBd
B->cd
