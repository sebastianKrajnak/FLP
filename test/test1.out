A,C
a
A
A->a
C->a
